tests/plugins/test_fetch_maven_artifacts.py

```diff
@@ -590,6 +642,32 @@ def test_fetch_maven_artifacts_url_bad_checksum(tmpdir, docker_tasker):
 
 
 @responses.activate
+def test_fetch_maven_artifacts_url_missing_checksum_value(tmpdir, docker_tasker):
+    """Err when remote file request does not provide checksum value."""
+    workflow = mock_workflow(tmpdir)
+    mock_koji_session()
+    mock_fetch_artifacts_by_url(str(tmpdir), contents=dedent("""\
+        - url: {root}/spam/spam.jar
+          sha1: ec61f019a3d0826c04ab20c55462aa2400000000
+          checksum_type: md5
+        """.format(root=FILER_ROOT)))
+    mock_url_downloads()
+    runner = PreBuildPluginsRunner(
+        docker_tasker,
+        workflow,
+        [{
+            'name': FetchMavenArtifactsPlugin.key,
+            'args': {'koji_hub': KOJI_HUB, 'koji_root': KOJI_ROOT}
+        }]
+    )
+
+    with pytest.raises(PluginFailedException) as e:
+        runner.run()
+
+    assert 'Missing checksum value attribute' in str(e)
+
+
+@responses.activate
```

Comment:
Stickler doesn't know how py.test fixtures work :(

This line should have `  # noqa` and a comment above that `docker_tasker` is a fixture